plugins/TagFix_Vatin.py

```diff
@@ -92,4 +98,3 @@ def test(self):
         assert a.node(None, {"ref:vatin": "ITAAAAAAAAAAA"})
         # missing country code
         assert a.node(None, {"ref:vatin": "11111111115"})
-
```

Comment:
Please, keep a blank line at the end.

Reply:
restored.. anyway it is a PEP8 standard, but I'm for the blank line too